plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -482,7 +481,7 @@ private TestExecutionResult testPluginAgainst(MavenCoordinates coreCoordinates,
 
             List<String> args = new ArrayList<String>();
             args.add("--define=maven.test.redirectTestOutputToFile=false");
-            args.add("--define=concurrency=1");
+            args.add("--define=forkCount=1");
```

Comment:
As of https://github.com/jenkinsci/plugin-pom/pull/80 (parent 2.37), `concurrency` is deprecated and `forkCount` is what you want. Noticed since a local profile

```xml
<profile>
    <id>faster</id>
    <activation>
        <activeByDefault>true</activeByDefault>
    </activation>
    <properties>
        <forkCount>1C</forkCount>
    </properties>
</profile>
```

was no longer being properly countermanded: multiple tests were running concurrently.